test/randomizer_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule RandomizerTest do
+  use ExUnit.Case
+
+  test "only returns numbers between 0 and the number provided" do
+    accumulated_results =
+      Enum.reduce(1..1000, [], fn _, results ->
+        {first_int, second_int} = Randomizer.random_coordinate_pair(10, 10)
+        results ++ [first_int, second_int]
+      end)
+      |> Enum.uniq()
+      |> Enum.sort()
+
+    assert(accumulated_results == [0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
+  end
```

Comment:
What do you guys think of this? It only adds another 0.01 seconds to the tests, but I'd love to hear about better ways of checking the results of randomization in tests.

Reply:
That's a great question! I can't think of a better way off the top of my head right now. One thing I will say is to be cautious of introducing too much logic into specs. This isn't overly complex but I've encountered scenarios where there was complex logic in specs to make an assertion but it turned out the logic being used was wrong which gave a false positive.

Reply:
I did some Googling and I wasn't able to find an alternative approach that I understood. Maybe Connor can also weigh in at some point. I don't think this is too expensive of a test! Great job benchmarking it!